doc/source/whatsnew/v1.4.0.rst

```diff
@@ -231,6 +231,7 @@ Other enhancements
 - :meth:`UInt64Index.map` now retains ``dtype`` where possible (:issue:`44609`)
 - :meth:`read_json` can now parse unsigned long long integers (:issue:`26068`)
 - :meth:`DataFrame.take` now raises a ``TypeError`` when passed a scalar for the indexer (:issue:`42875`)
+- Add support for `Zstandard <http://facebook.github.io/zstd/>`_ compression to :meth:`DataFrame.to_pickle`/:meth:`read_pickle` and friends (:issue:`43925`)
```

Comment:
"and friends" I know what you mean :) but I think there might be a better phrasing. Maybe "and most other IO functions". I think in the past I often mentioned only read_csv/to_csv as these are the most commonly used function.